VideoWeb/VideoWeb/Controllers/VideoEventsController.cs

```diff
@@ -73,14 +70,16 @@ public async Task<IActionResult> SendHearingEventAsync(ConferenceEventRequest re
                     request.ParticipantId = null;
                     events = request.CreateEventsForParticipantsInRoom(conference, roomId);
                 }
-
+                
                 var callbackEvents = events.Select(e => TransformAndMapRequest(e, conference)).ToList();
+
                 // DO NOT USE Task.WhenAll because the handlers are not thread safe and will overwrite Source<Variable> for each run
                 foreach (var e in events)
                 {
                     await SendEventToVideoApi(e);
                 }
 
+                callbackEvents.RemoveRepeatedVhoCallConferenceEvents();
```

Comment:
i would like to see a unit test for this to make sure VHOEvents are only published to the UI once

Reply:
This should be covered in the test should_not_call_multiple_callback_events_for_vho_call_events in SendVmrHearingEventsTests